tests/test_biolink_model/__snapshots__/biolink.graphql

```diff
@@ -1,5 +1,209 @@
 # metamodel_version: 1.7.0
 # version: 3.1.2
+enum AnatomicalContextQualifierEnum
+  {
+  }
+
+enum CausalMechanismQualifierEnum
+  {
+    binding
+    inhibition
+    antibody_inhibition
+    antagonism
+    molecular_channel_blockage
+    inverse_agonism
+    negative_allosteric_modulation
+    agonism
+    molecular_channel_opening
+    positive_allosteric_modulation
+    potentiation
+    activation
+    inducer
+    transcriptional_regulation
+    signaling_mediated_control
+    stabilization
+    stimulation
+    releasing_activity
+  }
+
+enum ChemicalEntityDerivativeEnum
+  {
+    metabolite
+  }
+
+enum ChemicalOrGeneOrGeneProductFormOrVariantEnum
+  {
+    genetic_variant_form
+    modified_form
+    loss_of_function_variant_form
+    gain_of_function_variant_form
+    polymorphic_form
+    snp_form
+    analog_form
+  }
+
+enum DirectionQualifierEnum
+  {
+    increased
+    upregulated
+    decreased
+    downregulated
+  }
+
+enum DrugAvailabilityEnum
+  {
+    over_the_counter
+    prescription
+  }
+
+enum DrugDeliveryEnum
+  {
+    inhalation
+    oral
+    absorption_through_the_skin
+    intravenous_injection
+  }
+
+enum DruggableGeneCategoryEnum
+  {
+    tclin
+    tbio
+    tchem
+    tdark
+  }
+
+enum FDAApprovalStatusEnum
+  {
+    discovery_and_development_phase
+    preclinical_research_phase
+    fda_clinical_research_phase
+    fda_review_phase_4
+    fda_post_market_safety_review
+    fda_clinical_research_phase_1
+    fda_clinical_research_phase_2
+    fda_clinical_research_phase_3
+    fda_clinical_research_phase_4
+    fda_fast_track
+    fda_breakthrough_therapy
+    fda_accelerated_approval
+    fda_priority_review
+    regular_fda_approval
+    post_approval_withdrawal
+  }
+
+enum FDAIDAAdverseEventEnum
+  {
+    life_threatening_adverse_event
+    serious_adverse_event
+    suspected_adverse_reaction
+    unexpected_adverse_event
+  }
+
+enum GeneOrGeneProductOrChemicalEntityAspectEnum
+  {
+    activity_or_abundance
+    abundance
+    activity
+    expression
+    synthesis
+    degradation
+    cleavage
+    hydrolysis
+    metabolic_processing
+    mutation_rate
+    stability
+    folding
+    localization
+    transport
+    secretion
+    uptake
+    molecular_modification
+    acetylation
+    acylation
+    alkylation
+    amination
+    carbamoylation
+    ethylation
+    glutathionylation
+    glycation
+    glycosylation
+    glucuronidation
+    n_linked_glycosylation
+    o_linked_glycosylation
+    hydroxylation
+    lipidation
+    farnesylation
+    geranoylation
+    myristoylation
+    palmitoylation
+    prenylation
+    methylation
+    nitrosation
+    nucleotidylation
+    phosphorylation
+    ribosylation
+    ADP-ribosylation
+    sulfation
+    sumoylation
+    ubiquitination
+    oxidation
+    reduction
+    carboxylation
+  }
+
+enum GeneOrGeneProductOrChemicalPartQualifierEnum
+  {
+    3_prime_utr
```

Comment:
I will have a look at it. Assuming such a name is not allowed in GraphQL, the `_format_name` function [you have proposed](https://github.com/linkml/linkml/pull/2303#discussion_r1751051912) should fix it.

Reply:
truly have no idea what the best option here would be - as long as there is some mapping from linkml -> graphql that can be inspected/documented i think it's fine. erroring saying "graphql can't support this name" should probably come with a bit of a deprecation period since people might be using the (incorrect?) schemas anyway already.

Reply:
Not relevant anymore. Addressed on later changes.